CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ Last releases: [[1.9.0] - 2019-05-22](#190---2019-05-22) and [[1.8.0] - 2019-04-
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
```

Comment:
As per https://github.com/math-comp/math-comp/wiki/Agenda-of-the-April-23rd-2019-meeting-9h30-to-12h30#avoiding-issues-with-changelog new changelog entries now go to `CHANGELOG_UNRELEASED.md`

Reply:
I created the file, keeping `## [Unreleased]` section, which looks a bit weird since the whole file is devoted to unreleased stuff, but I think it should make it easier to move entries from `CHANGELOG_UNRELEASED.md` to `CHANGELOG.md`. I can change that if you'd like, though.

Reply:
I think this is fine